deploy/scripts/install-combine.sh

```
@@ -325,6 +353,24 @@ if [ "${STATE}" != "Uninstall-combine" ] ; then
     echo "Helm timeout: ${HELM_TIMEOUT}"
     SETUP_OPTS="--timeout ${HELM_TIMEOUT}"
   fi
+  # Fonts are installed in the Install-combine step, which also runs after the
+  # restart, so record the language list and reuse it until the install finishes.
+  if [ -z "${LANGS}" ] ; then
+    if [ -f ${LANGS_FILE} ] ; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)